types/rc-tooltip/index.d.ts

```diff
@@ -31,7 +32,7 @@ declare namespace RCTooltip {
 		placement?: Placement | Object;
 		align?: Object;
 		onPopupAlign?: (popupDomNode: Element, align: Object) => void;
-		overlay: React.ReactNode;
+		overlay: React.ReactChild | React.ReactFragment | React.ReactPortal;
```

Comment:
From [documentation](https://github.com/react-component/tooltip#props), I think function type should be included in this union.

> `React.Element | () => React.Element`

Reply:
Great catch! Addressed!